### DISCUSSION TITLE
JellyFin Noob here

### DISCUSSION THREAD

**Discussion by wizard-zoso** [jellyfin/jellyfin#5742] [Q&A]
Perhaps Someone can help me with a few things I am using windows 1064bit,
i use to use plex for a few years and still keep it as a backup as i do not know if any of these Emby or Jelly can run if the internet goes out like Plex can i moved on to Emby server. for the time being

There are features that Emby has that Jellyfin don't have or maybe there are settings i do not know about to reveal them.

1) For example lets say i have a season of 30 episodes i can keep all 30 on one page without having to choose page --> 2
2) when i choose and click Favorites they are not as easy to navigate as i Emby
3) the first main page is not transparent as i would like it to be the Main page server

I have tried the codes but they don't seem to apply to jellyfin on the main page but only in the sub folder.

I really want to like Jelly more then Emby but i am still a noob and hope someone can get me in the right direction starting with my Episodes  on more then one page problem? i want to keep them on one page, can we start there?

Thank You

**Comment by belfour** [comment]
"keep all 30 on one page without having to choose page --> 2"

Try this.  In the browser click on the account icon (top right) --> Display --> Library Page Size.  This sets the amount of items to show on a library page. Set to 0 in order to disable paging. 

I don't use Favorites so can't help you there.  I doubt transparency is a high priority but you might be able to play with CSS

    background-color: transparent!important;

**Reply by wizard-zoso** [reply]
Thank You so much for this
I am hoping this will also affect when i Use JellyFin client on the Roku?
